lib/harness/manifest.ex

```diff
@@ -65,6 +82,7 @@ defmodule Harness.Manifest do
     :ok =
       deps
       |> Enum.flat_map(&Mix.Dep.load_paths/1)
+      |> (&[archive_path() | &1]).()
```

Comment:
Is this just appending the archive path to the list?  Hard to read IMO.

Reply:
yup, it's basically
```
|> (fn path_list -> [archive_path() | path_list] end).()
```
but I thought to use the shorter version. I can write it up in the long form if needed

Reply:
Meh, it's fine.  If we were doing it a lot maybe.  Do you need to reverse this list afterward or does it matter?  

If you were doing this kind of thing 3+ times I would prefer to see a function *like* this:
https://elixirforum.com/t/append-to-list-after-enum-map-how-to-do-this-efficiently/47626/10

Reply:
totally agree with you there, as it's just a one-off I went with this little cutie :)

and the order doesn't matter so I opted to prepend to the list since it's done in constant time (ie. no traversal needed)